docs-content/using/deploy-app.html.md.erb

```diff
@@ -0,0 +1,211 @@
+* Line 4 specifies a fully qualified secret ID, and indicates that the value of the secret should be written to a temp file and `SSL_CERT` should be set to the temp file path.
+
+In the context of a PCF environment, the above example could produce the following results, with the first two values and the contents of the file retrieved from Conjur:
+
+```
+DB_USERNAME: AKIAI44QH8DHBEXAMPLE
+DB_PASSWORD: je7MtGbClwBF/2Zp9Utk/h3yCo8nvbEXAMPLEKEY
+REGION: us-east-1
+SSL_CERT: pcf/tmp/ssl-cert.pem
+```
+
+For more information about syntax and options for a `secrets.yml` file, see the `secrets.yml` section in the [Summon documentation](https://cyberark.github.io/summon/).
+
+##### Example: my-app.yml
+
+The following example shows a Conjur application policy declaring two secrets as variables for Conjur to manage and a group with access to those secrets.
+
+```
+- !policy
+  id: my-app
+  body:
+    - &variables
+      - !variable db/username
+      - !variable db/password
+
+    - !group secrets-users
+
+    # secrets-users can read and execute
+    - !permit
+      resource: *variables
+      privileges: [ read, execute ]
+      role: !group secrets-users
+```
+
+Values can be stored in these variables using the Conjur CLI, for example:
+
+```
+$ conjur variable values add "my-app/db/username" "My username value"
+Value added
+```
+
+#### <a id="create-service"></a> Step 2: Create a Service Instance
+
+In the PCF space where you intend to deploy the application, create a service instance.
+
+<pre class='term'>$ cf create-service cyberark-conjur community conjur</pre>
+
+In the command above:
+
+* The service-name is `cyberark-conjur`.
+* There is a single free service plan called `community`.
+* For convenience, you may use the same instance-name (e.g. `conjur`) in multiple PCF spaces,
+  so that the same application manifest works in all spaces. The instance name may be any
+  value you choose, and does not have to be `conjur`, but must match the service name you
+  bind applications in that space to.
+
+If you do not yet have a space for the application, you may create a one with:
+
+```
+cf create-space "My Space"
+```
+
+And then target the space with:
+
+```
+cf target -s "My Space"
+```
+
+#### <a id="permit-org-space"></a> Step 3: Permit Secret Access by Org and Space (Conjur Open Source and Conjur EE v5+ only)
+
+Adding a service instance to a space will automatically create resources within Conjur
+to allow privileging all of the apps in that org or space to access a secret in Conjur. These
+resources are Conjur layers named using the Globally Unique Identifiers (GUIDs) for the Org and
+Space.
+
+The Org and Space GUIDs may be retrieved from CloudFoundry using the `cf` CLI tool. For example:
+```
+$ cf org --guid my-org
+8f366b77-d826-4522-8cc6-261bf9b02cba
+
+$ cf space --guid my-space
+4b02b69c-ba35-4069-b634-ded5644740c9
+```
+
+Assuming the Service Broker policy branch is `pcf/production`, the org and space layers
```

Comment:
If  the Service Broker policy branch is `pcf/production`, the org and space layers